src/tuning/visualize_tuning.py

```
@@ -0,0 +1,857 @@
+
+        plt.tight_layout()
+        plt.subplots_adjust(top=0.93)
+
+        # Save
+        output_path = PLOTS_DIR / "dashboard1_best_model_recommendations.png"
+        plt.savefig(output_path, dpi=300, bbox_inches="tight")
+        plt.close()
+
+        logger.info(f"  ✅ Dashboard 1 saved to {output_path}")
+
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)